docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -35,14 +35,32 @@ You can specify more than one SStable.
 
 Schema
 ------
+
 All operations need a schema to interpret the SStables with.
-Currently, there are two ways to obtain the schema:
+The tool tries to auto-detect the location of the scylla data directories and the name of the table the sstable belongs to.
+If the sstable is located in a scylla data directory, this works out-of-the box, without any additional input from the user.
+If the sstable is located at an external path, the tool has to be told the name of the keyspace and table the sstable belongs to. And potentially some hints as to where the scylla data dir is located.
+
+Currently, the schema can be obtained in the following ways:
 
+* Auto detect - if the sstable is located in the table's directory, within the scylla data directory.
+* ``--keyspace=KEYSPACE --table=TABLE`` - if the sstable is located at an external location, but the scylla data directory or the config file are located at standard location. The tool also reads the ``SCYLLA_CONF`` and ``SCYLLA_HOME`` environment variables to try to locate the configuration file.
 * ``--schema-file FILENAME`` - Read the schema definition from a file.
-* ``--system-schema KEYSPACE.TABLE`` - Use the known definition of built-in tables (only works for system tables).
+* ``--system-schema --keyspace=KEYSPACE --table=TABLE`` - Use the known definition of built-in tables (only works for system tables).
+* ``--scylla-data-dir SCYLLA_DATA_DIR_PATH --keyspace=KEYSPACE --table=TABLE`` - Read the schema tables from the data directory at the provided location, needs the keyspace and table name to be provided with ``--keyspace`` and ``--table``.
+* ``--scylla-yaml-file SCYLLA_YAML_FILE_PATH --keyspace=KEYSPACE --table=TABLE`` - Read the schema tables from the data directory path obtained from the configuration, needs the keyspace and table name to be provided with ``--keyspace`` and ``--table``.
+
+By default (no schema related options are provided) the tool will try the following squence:
```

Comment:
```suggestion
By default (no schema-related options are provided), the tool will try the following sequence:

```

Reply:
Apart from the language fixes, this suggestion adds an empty line between the text and the list. Without the line, the list doesn't render (it's a long line of text instead of bullets on separate lines).